tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -77,7 +77,7 @@ int es = PAPI_NULL; //event set
 
 size_t NB_ITER = CONST_NB_ITER;
 size_t M = WG_MULT0*J_CL_DEVICE_CONST_WORK_GROUP_SIZE0+JACOBI_DELTA;
-size_t N = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
+size_t K = WG_MULT1*J_CL_DEVICE_CONST_WORK_GROUP_SIZE1+JACOBI_DELTA;
```

Comment:
Curious this change...

Is this an automatic Visual Studio feature? :-)

In the following, in the same way we have `(i, j)`, it is common to have `(M, N)` for the size instead of `(M, K)`.
Perhaps I touched too much Fortran 77 in the past... :-)

The same in all the Jacobi tests.